docker-compose.yml

```diff
@@ -1,5 +1,49 @@
 version: "3"
 services:
+  celestia-devnet:
+    container_name: celestia_devnet
+    image: ghcr.io/rollkit/local-celestia-devnet:v0.11.0-rc12
```

Comment:
Bump this to latest
https://github.com/rollkit/local-celestia-devnet/releases/tag/v0.12.1